.changeset/pre.json

```diff
@@ -0,0 +1,10 @@
+{
+  "mode": "pre",
+  "tag": "beta",
+  "initialVersions": {
+    "docs-app": "0.0.0",
+    "@crowdstrike/ember-toucan-core": "0.0.0",
```

Comment:
This depends on the answer to the previous question, but just to point this out: the `0.0.x` releases are a bit tricky and awkward IMO, as according to how npm version modifiers work any new patch release like `0.0.x+1` is considered a breaking change, as `^0.0.1` would _not_ allow `0.0.2` to be used. So unless our API is really like super unstable, I would start with `0.1.0` (as `^0.1.0` would cover all `0.1.x` releases then).